packages/main/src/InputPopover.hbs

```diff
@@ -24,15 +24,15 @@
 			</div>
 			<div class="row">
 				<div class="input-root-phone">
-					<input
+					<ui5-input
 						class="ui5-input-inner-phone"
 						type="{{inputType}}"
 						.value="{{value}}"
-						inner-input
+						.show-clear-icon={{showClearIcon}}
```

Comment:
Now what happens is the opposite - the clear icon is forwarded to the main input, but no longer shows in the dialog.

Removing the period seems to bind the property correctly:

```suggestion
						show-clear-icon={{showClearIcon}}
```